setup.cfg

```diff
@@ -0,0 +1,10 @@
+[flake8]
+max-line-length = 90
+max-complexity = 10
+statistics = true
+application-import-names = user_deletion
+import-order-style = smarkets
```

Comment:
`smarkets` has been added in https://github.com/public/flake8-import-order/pull/57

> smarkets - style as google only with import statements before from X import